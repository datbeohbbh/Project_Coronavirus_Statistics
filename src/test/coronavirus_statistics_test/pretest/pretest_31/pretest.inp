50 100
? 5
? 5
? 5
? 5
q 1 27 209
? 4 1
q 1 8 418
q 1 21 331
q 1 22 425
q 1 5 140
? 4 2
? 4 2
q 1 44 118
? 7
? 4 4
? 4 2
q 1 50 328
q 1 17 79
? 4 1
q 1 45 408
q 1 33 296
q 1 36 66
q 1 15 209
? 4 4
? 4 9
? 4 10
q 1 48 34
? 7
? 4 2
q 1 35 422
q 2 35 318
q 1 29 165
q 3 21 95
q 3 21 228
? 6
q 3 17 76
q 1 30 270
q 1 6 161
q 1 41 339
q 2 36 56
? 4 4
q 2 36 8
q 3 22 337
q 2 50 21
? 4 2
q 3 22 57
q 2 8 305
q 1 34 19
q 1 25 412
? 7
q 1 13 249
q 2 30 188
q 1 22 140
q 3 50 27
? 7
q 1 22 486
? 6
? 6
? 4 13
q 3 33 162
q 1 7 413
? 4 6
q 2 30 34
q 3 25 12
q 1 38 222
? 6
q 1 45 331
? 4 16
q 1 1 471
q 1 4 464
? 4 5
? 4 5
q 3 34 2
q 3 38 164
q 3 45 410
q 1 14 216
q 3 35 42
? 4 3
? 4 1
? 4 7
q 2 8 56
? 4 17
q 1 18 87
? 6
? 4 13
? 4 22
? 4 12
? 4 19
? 7
? 6
? 7
? 4 2
? 4 7
? 4 2
? 4 24
? 4 25
? 4 4
? 4 17
? 4 14
? 6