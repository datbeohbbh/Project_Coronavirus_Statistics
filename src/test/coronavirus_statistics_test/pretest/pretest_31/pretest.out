Query ? 5 :
The number of safe cities is 50

Query ? 5 :
The number of safe cities is 50

Query ? 5 :
The number of safe cities is 50

Query ? 5 :
The number of safe cities is 50

Query ? 4 1 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 209
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 209
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 209
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 4 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 331
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 140
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 79
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 140
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 331
-> Has recovered : 0
-> Was dead : 0

Query ? 4 10 :
The answer is the city with the index 45 
- Information of the city 45 -
-> Number of current patients : 408
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 2 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 66
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 318 people

Query ? 4 4 :
The answer is the city with the index 48 
- Information of the city 48 -
-> Number of current patients : 34
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 3
-> Has recovered : 0
-> Was dead : 76

Query ? 7 :
The city with the highest death is city with the index 22
Quantity: 394 people

Query ? 7 :
The city with the highest death is city with the index 22
Quantity: 394 people

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 318 people

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 318 people

Query ? 4 13 :
The answer is the city with the index 15 
- Information of the city 15 -
-> Number of current patients : 209
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 30 
- Information of the city 30 -
-> Number of current patients : 82
-> Has recovered : 188
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 35
Quantity: 318 people

Query ? 4 16 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 222
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 48 
- Information of the city 48 -
-> Number of current patients : 34
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 48 
- Information of the city 48 -
-> Number of current patients : 34
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 8
-> Has recovered : 0
-> Was dead : 323

Query ? 4 1 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 2
-> Has recovered : 64
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 38 
- Information of the city 38 -
-> Number of current patients : 58
-> Has recovered : 0
-> Was dead : 164

Query ? 4 17 :
The answer is the city with the index 14 
- Information of the city 14 -
-> Number of current patients : 216
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 8
Quantity: 361 people

Query ? 4 13 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 140
-> Has recovered : 0
-> Was dead : 0

Query ? 4 22 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 339
-> Has recovered : 0
-> Was dead : 0

Query ? 4 12 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 134
-> Has recovered : 0
-> Was dead : 162

Query ? 4 19 :
The answer is the city with the index 13 
- Information of the city 13 -
-> Number of current patients : 249
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 45
Quantity: 410 people

Query ? 6 :
The city with the highest recovery is city with the index 8
Quantity: 361 people

Query ? 7 :
The city with the highest death is city with the index 45
Quantity: 410 people

Query ? 4 2 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 3
-> Has recovered : 0
-> Was dead : 76

Query ? 4 7 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 57
-> Has recovered : 361
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 3
-> Has recovered : 0
-> Was dead : 76

Query ? 4 24 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 413
-> Has recovered : 0
-> Was dead : 0

Query ? 4 25 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 464
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 17
-> Has recovered : 0
-> Was dead : 2

Query ? 4 17 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 209
-> Has recovered : 0
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 161
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 8
Quantity: 361 people

